### ISSUE TITLE
Let's select a paper for our paper reading/discussion session on May 30, 2020

### ISSUE THREAD

**Issue by omarsar** [dair-ai/nlp-paper-discussions#1]
Comment a paper you would like us to discuss during our weekly paper reading discussion. 

You can vote on a suggested paper by using the 👍 emoji. I will close the issue in one or two days and select the paper with the most votes. Then I will make the announcement at the beginning of the week. Thanks.

**Comment by omarsar** [comment]
Title: Oscar: Object-Semantics Aligned Pre-training for Vision-Language Tasks 
Link: https://arxiv.org/abs/2004.06165

**Comment by BennoKrojer** [comment]
Title: Experience Grounds Language
Link: https://arxiv.org/abs/2004.10151

**Comment by shashankhalo7** [comment]
Title : Human Instruction-Following with Deep Reinforcement Learning via Transfer-Learning from Text
Paper : https://arxiv.org/abs/2005.09382

**Comment by manisnesan** [comment]
**Title**: A brief introduction to weakly supervised learning
**Paper**: https://api.semanticscholar.org/CorpusID:44192968 
**Taxonomy**: Modeling -> Training -> Weak Supervised Learning 

Why: Modeling topic, literature review type, and also practical in nature.  Taxonomy is obtained from 
https://nlusense.com/v/32:18

**Comment by vsistla** [comment]
Title - AI Trust in business processes: The need for process-aware explanations
Summary - Business processes underpin a large number of enterprise operations including processing loan applications, managing invoices, and insurance claims. There is a large opportunity for infusing AI to reduce cost or provide better customer experience, and the business process management (BPM) literature is rich in machine learning solutions including unsupervised learning to gain insights on clusters of process traces, classification models to predict the outcomes, duration, or paths of partial process traces, extracting business process from documents, and models to recommend how to optimize a business process or navigate decision points. More recently, deep learning models including those from the NLP domain have been applied to process predictions.

Unfortunately, very little of these innovations have been applied and adopted by enterprise companies. We assert that a large reason for the lack of adoption of AI models in BPM is that business users are risk-averse and do not implicitly trust AI models. There has, unfortunately, been little attention paid to explaining model predictions to business users with process context. We challenge the BPM community to build on the AI interpretability literature, and the AI Trust community to understand

Paper - https://arxiv.org/abs/2001.07537
Submission history
From: Vinod Muthusamy [view email]
[v1] Tue, 21 Jan 2020 13:51:36 UTC (545 KB)

**Comment by goodhamgupta** [comment]
**Title**: The Probabilistic Relevance Framework: BM25 and Beyond
**Paper**: [Github](https://github.com/tpn/pdfs/blob/master/The%20Probabilistic%20Relevance%20Framework%20-%20BM25%20and%20Beyond.pdf)
**Summary**: This paper talks about the origins of BM25 but more importantly details out a mathematical framework to incorporate custom features while scoring and fetching relevant documents. BM25 is also the default scoring metric used in ElasticSearch.
**Resources**: 
- Fantastic summary about BM25 by Dr. Britta Weber is available [here](https://www.youtube.com/watch?v=v3Ko0CwgTZ0)
- Probabilistic Models for IR short videos by Dr. Victor Lavrenko available [here](https://www.youtube.com/playlist?list=PLBv09BD7ez_56YwzSyQOOsrAE4Js7XuAB)
- Similarity module by ElasticSearch available [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-similarity.html) which shows the other evaluation metrics present such as TF-IDF, DFI(Divergence from Independence),  etc.

**Comment by msank00** [comment]
**Title: Colors in Context: A Pragmatic Neural Model for
Grounded Language Understanding**
- paper [link](https://www.aclweb.org/anthology/Q17-1023/)
- EMNLP2017

**Why:** This paper is related to *Language grounding*

> How does language relate to the real world?

For example,
- How do we map visual data onto colour terms such as “red” :red_circle:
- How do we map clock times onto time expressions such as “late evening”?
- How do we map geographic regions onto descriptors such as “central Scotland”?

**Grounded language** acquisition is concerned with learning the meaning of language as it applies to the physical world. As robots become more capable and ubiquitous, there is an increasing need for non-specialists to interact with and control them, and natural language is an intuitive, flexible, and customizable mechanism for such communication.

:paperclip: **Resources:**
- [AAAI2013 Talk on Grounded Language Learning](http://videolectures.net/aaai2013_mooney_language_learning/)

**Comment by omarsar** [comment]
Hi all! Thanks for suggesting the papers. There were some really interesting ones listed here. Unfortunately, we can't select them all but you have the chance to bring it up again in the future. The winning paper is:

Title: Experience Grounds Language
Link: https://arxiv.org/abs/2004.10151

I will make the official announcement shortly.
